assets/js/sticky-menu.js

```diff
@@ -147,6 +148,11 @@ define( ['jquery', 'underscore'], function ( $, _ ) {
 		/**
 		 * Is the direction of the scroll = up?
 		 */
+		isAllDirectionEnabled: function () {
+			// Return true, if scroll all direction is set
```

Comment:
@krumpak This comment is not needed inside the method, since it's just a one line method, so this can be removed.